internal/util/reports_test.go

```diff
@@ -12,9 +14,65 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type TLSPair struct {
+	Key, Cert string
+}
+
+var (
+	tlsPair = TLSPair{
```

Comment:
I think a rename of this variable might be helpful and informative to new code readers at a glance:

```suggestion
	tlsCertForTestingReports = TLSPair{
```

I'm OK if you want to change that up, that just reads really well to me and I know exact what this cert is for at a glance.